vite-plugin/src/query.js

```diff
@@ -33,7 +33,7 @@ export function parseViteRequest (id) {
           query.type === 'template' ||
           // On prod, TS code turns into a separate 'script' request.
           // See: https://github.com/vitejs/vite/pull/7909
-          (query.type === 'script' && query['lang.ts'] !== void 0),
+          (query.type === 'script' && isOfExt({ query, extensions: ['.ts', '.tsx'] })),
```

Comment:
I think it should be `&& (query['lang.ts'] !== void 0 || query['lang.tsx'] !== void 0)

Reply:
`isOfExt` without `filename` does the same thing, doesn't it?

Reply:
It's all about speed of execution. This needs to be as fast as possible.